### DISCUSSION TITLE
Is the latent (with leftover noise) from SDXL incompatible with SD1.5 ?

### DISCUSSION THREAD

**Discussion by ntdviet** [Comfy-Org/ComfyUI#1098] [Q&A] [answered]
I tried to use the SDXL latent image (with leftover noise) in a sampler using SD1.5 model and it yield a mess.
Latent composite using SDXL and SD1.5 latents also doesn't work when there is leftover noise :(

It worked well for purely SDXL workflows, or purely SD1.5. The mix doesn't work for me.
Did I do something wrong or is there a workaround?

**Comment by comfyanonymous** [comment, marked as answer]
The latent images are not compatible between SD1.x and SDXL. You have to either VAEDecode and VAEEncode or use something like: https://github.com/city96/SD-Latent-Interposer

**Reply by ntdviet** [reply]
Great, thanks!

**Reply by ntdviet** [reply]
I used the SD-Latent-Interposer to join SDXL latents into SD1.x pipelines. It works like a charm. Again, many thanks
